### DISCUSSION TITLE
How to exit element zapper mode in mobile phone

### DISCUSSION THREAD

**Discussion by peace2000** [uBlockOrigin/uBlock-issues#1815] [Q&A] [answered]
How do you actually exit the element zapper mode in a mobile phone? When I activate that mode and zap the wanted element, zapper mode still stays active and tries to zap new elements in the current web page...

**Comment by peace2000** [comment, marked as answer]
Oh nevermind, found the answer https://github.com/uBlockOrigin/uBlock-issues/wiki/Element-zapper

Though that's something that's not easy to figure out without reading the wiki.

**Comment by Ian-Arlen** [comment]
This is an astonishingly bad mechanism. Swipes arent reliable. There really ought to be a simpler way to handle this. (press and hold anywhere?)

**Reply by kupiakos** [reply]
If it's possible to intercept the Android back button/gesture, that seems like a good choice.

**Reply by garry-ut99** [reply]
Also see here:
- https://github.com/uBlockOrigin/uBlock-issues/issues/3268
- https://github.com/uBlockOrigin/uBlock-issues/issues/1968
